pkgs/ffigen/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 16.0.1-wip
+## 17.0.0-wip
```

Comment:
What's this change for? Can we make it 16.1 instead?

Reply:
It's because `ffigen` has breaking changes, see the PR health comment. Happy to disable the breaking change check for `ffigen` in general, address the upgrade in a later PR, or just ignore this breakage.

Reply:
I think the issue is that the breaking change checker is checking for changes on internal classes. This is the only breaking change in the report:

```json
  "breakingChanges": {
    "label": "BREAKING CHANGES",
    "children": [
      {
        "label": "Class ImportedType",
        "children": [
          {
            "label": "Constructor ImportedType",
            "children": [
              {
                "changeDescription": "Kind of parameter \"defaultValue\" changed. not named -> named",
                "changeCode": "CE07",
                "isBreaking": true,
                "type": "major"
              }
            ]
          }
        ]
      }
    ]
  },
```

`ImportedType` is not part of the public API (or at least, it shouldn't be public). The fix here is to figure out why that class (and others mentioned in the non-breaking changes) are considered to be part of the public API. It's probably related to the issues in the API leaks check. But if you want to just ignore ffigen for now, that's fine.

Reply:
Right - the leaks could have something to do with that. Those clean ups should be done in follow ups.